### PR TITLE
[RHELC-1102] Enable el9 conversions config updates

### PR DESCRIPTION
**Pull request by pr-watson**
This PR contains updates to config files, yaml files, the spec file to enable el9 conversions taken from the larger PR - https://github.com/oamg/convert2rhel/pull/883. 

Until we get confirmation that everything works fine we will keep this in draft.
Jira Issues: [RHELC-1102](https://issues.redhat.com/browse/RHELC-1102)

Checklist

- [ ] PR has been tested manually in a VM (either author or reviewer)
- [ ] Jira issue has been made public if possible
- [x] `[RHELC-]` is part of the PR title 
- [x] GitHub label has been added to help with Release notes 
- [x] PR title explains the change from the user's point of view
- [x] Code and tests are documented properly
- [x] The commits are squashed to as few commits as possible (without losing data) 
- [ ] When merged: Jira issue has been updated to `Release Pending` if relevant

### CODE REVIEW

**Review by SpyTec — APPROVED** [review]

File: packaging/convert2rhel.spec [hunk 1]
```diff
@@ -67,8 +67,9 @@ Requires:       grub2-tools
 The purpose of the convert2rhel tool is to provide an automated way of
 converting the installed other-than-RHEL OS distribution to Red Hat Enterprise
 Linux (RHEL). The tool replaces all the original OS-signed packages with the
-RHEL ones. Available are conversions of CentOS Linux 7/8, Oracle Linux 7/8
-and Scientific Linux 7 to the respective major version of RHEL.
+RHEL ones. Available are conversions of Scientific Linux 7, CentOS Linux 7/8,
+Oracle Linux 7/8/9, Alma Linux 8/9, and Rocky Linux 8/9 to the respective major
+version of RHEL.
```
Comment (SpyTec) [inline packaging/convert2rhel.spec:72]: Until we get it working lets skip this for now
```suggestion
RHEL ones. Available are conversions of CentOS Linux 7/8, Oracle Linux 7/8
and Scientific Linux 7 to the respective major version of RHEL.
```

File: convert2rhel/data/9/x86_64/configs/oracle-9-x86_64.cfg [hunk 2]
```diff
@@ -0,0 +1,41 @@
+[system_info]
+
+# Fingerprints of GPG keys used for signing packages of the OS to be converted.
+# The GPG key is available at http://public-yum.oracle.com/RPM-GPG-KEY-oracle-ol9
+# Delimited by whitespace(s).
+gpg_fingerprints = bc4d06a08d8b756f a7dd07088b4efbe6
```
Comment (r0x0d) [inline convert2rhel/data/9/x86_64/configs/oracle-9-x86_64.cfg:6]: Can we put them in their own line? 

```suggestion
gpg_fingerprints = 
  bc4d06a08d8b756f 
  a7dd07088b4efbe6
```

Comment (pr-watson) [reply convert2rhel/data/9/x86_64/configs/oracle-9-x86_64.cfg]: Sure!

Comment (r0x0d) [reply convert2rhel/data/9/x86_64/configs/oracle-9-x86_64.cfg:6]: I don't see the changes here, @pr-watson.

Comment (pr-watson) [reply convert2rhel/data/9/x86_64/configs/oracle-9-x86_64.cfg:6]: Weird, I did commit suggestion, let me manually do it

**Review by SpyTec — APPROVED** [review]

**Pull request merged**